### PR TITLE
ci(Travis): defaults travis build

### PR DESCRIPTION
**Pull request by d3viant0ne**
- Adds standard travis build file
- updates travis:lint to execute nsp check

### CODE REVIEW

**Comment by bebraw** [comment]
About Bithound, would it be possible to enable the checks per repo through a cli when `webpack-defaults` is attached to the project? I had to fiddle with the UI yesterday to make it work for **file-loader**. I'm pretty sure we don't want to do that for each repository separately. 👍

**Comment by d3viant0ne** [comment]
I'll have to look and see if they have a rest API though given it involves adding a webhook, i'm betting no.

It's not the only 3rd party service that requires a bit of initialization. Codecov has to be manually spun up once as well.

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "test:coverage": "jest --collectCoverageFrom='src/**/*.js' --coverage",
     "travis:test": "yarn run test",
     "travis:coverage": "yarn run test:coverage",
-    "travis:lint": "yarn run lint && yarn run nsp",
+    "travis:lint": "yarn run lint && yarn run security",
```
Comment (sapegin) [inline package.json]: It should be fixed in `src/tasks/package.js` too.

**Comment by sapegin** [comment]
Why not Node 4.3? The latest mrm-core will work in 4.3.

**Pull request merged**